src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -51,13 +51,17 @@ export const benefitsRelinquishedWarning = (
 );
 
 export const benefitsDisclaimerSpouse = (
-  <p>
-    IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). <strong>You cannot retain eligibility for both programs simultaneously</strong>.
-  </p>
+  <div className="usa-alert usa-alert-info">
```

Comment:
@goldenmeanie @peggygannon I see you ultimately decided to leave this as is to make it more subdued (in your discussion in department-of-veterans-affairs/vets.gov-team#2867). I will revert.